### PR TITLE
Style: Change esthetic -> aesthetic

### PR DESCRIPTION
**Pull request by ericrallen**
This PR ensures consistent spelling for `aesthetic` across the in-editor docs (and also a few in-code comments where it is written), preferring the more widely-used "ae" spelling.

### CODE REVIEW

**Review by AThousandShips — CHANGES_REQUESTED** [review]
This is not correct, esthetic is the US English spelling, aesthetic is generally the UK one, and we use the US spelling when it differs

Source: https://en.wiktionary.org/wiki/esthetic#English

**Comment by ericrallen** [comment]
Happy to change it to the `e` if that's really preferred, but I don't think most people actually use the `e` spelling unless they are referring to skincare, where esthetic and esthetician are popular spellings.

It also seems pretty rare to find anything written about video games, art, or design that doesn't use the `ae` spelling.

As a US-born, native English speaker, I had to Google "estethic" based on your response and this English StackExchange answer has some interesting deep dives into this: https://english.stackexchange.com/q/173585

I just noticed it spelled differently in two places while looking at some in-editor tooltips.

**Comment by AThousandShips** [comment]
I'd say to leave it be, it's a minor difference either way

File: doc/translations/zh_CN.po
Comment (AThousandShips) [inline doc/translations/zh_CN.po:1]: This file shouldn't be edited

File: doc/translations/zh_TW.po
Comment (AThousandShips) [inline doc/translations/zh_TW.po:1]: Or this one

**Review by AThousandShips — DISMISSED** [review]
We can keep this for discussion, I don't think this change is worth it as it's a different (yet standard) spelling more US specific which fits our guidelines, though I don't think we should change `aesthetic` to `esthetic` either

If you still want this change please fix the translation files, they should not be manually edited, and after that I'll remove the changes requested and we'll have this as up for discussion

**Comment by LinuxUserGD** [comment]
> This is not correct, esthetic is the US English spelling, aesthetic is generally the UK one, and we use the US spelling when it differs
> 
> Source: https://en.wiktionary.org/wiki/esthetic#English

See https://www.merriam-webster.com/dictionary/aesthetic (comment: https://forum.wordreference.com/threads/aesthetic-esthetic.1502784/#post-7604976).
Both variants seem to be equally common in US English spelling.

File: doc/translations/zh_CN.po
Comment (ericrallen) [reply doc/translations/zh_CN.po:1]: 👍 Sorry about that. Should have been more targetted in my replacing.

File: doc/translations/zh_TW.po
Comment (ericrallen) [reply doc/translations/zh_TW.po:1]: 👍 Sorry about that. Should have been more targetted in my replacing.

**Comment by ericrallen** [comment]
I removed the changes to the translation files. Sorry for the confusion on that.

If this is too minor a change, feel free to close it - I know it's not actually that important.

I just noticed discrepancies in how the word was spelled between the official docs and then in a few places in the in-editor tooltips and wanted to make it consistent - and `aesthetic` is the far more common spelling, especially when referring to visual arts.

**Comment by Mickeon** [comment]
There's actually a pretty fair debate on how people expect this word to be spelled as, so keep this PR open. At most it will just not ever be merged.
Note that it's difficult to justify these fixes in the docs unless they are typos or plainly inaccurate, because it causes the corresponding translations to become "outdated" even with minor changes.

It's not as clear-cut case of "esthetic is the `en_us` way of saying it" because both are completely fine even in US English. Now it's a matter of questioning how **non-native speakers** expect it to be.
One of the lines this PR fixes was even written by me as "aesthetic" originally, before @AThousandShips suggested to change it.

**Comment by akien-mga** [comment]
It's anecdotal, but as a non native speaker "aesthetic" is a lot more natural to me.

codespell had the replacement in their en_GB to en_US dictionary for a bit, but removed it as it was indeed pointed out that aesthetic is valid in en_US.

I think we can just pick one and keep it consistent, I don't think this would rub people off the wrong way. Since aesthetic is ok in en_US, I'd say let's be nice to en_GB and international speakers by picking the "ae" variant.

**Review by Mickeon — APPROVED** [review]
Non-native speaker and it's my own preference too.

**Review by AThousandShips — APPROVED** [review]
Don't have a preference either way but this standardized things

**Review by marcospb19 — APPROVED** [review]
non-native here too, I didn't know "esthetic" existed and meant "aesthetic", I'm familiar with the latter

**Review by aaronfranke — APPROVED** [review]
I'm from the US, native English speaker, and I've always seen "aesthetic", never any other version. The existence of "esthetic" is brand new information to me. I say let's go with the correct and standard "aesthetic".

**Comment by mrjustaguy** [comment]
As a non native speaker, I've only ever heard of aesthetic, and at first esthetic seemed to me like a typo

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**